python/ray/dag/tests/test_accelerated_dag.py

```diff
@@ -48,6 +49,13 @@ def inc_two(self, x, y):
         self.i += y
         return self.i
 
+    def inc_four(self, a, b, x, y):
+        self.i += a
+        self.i += b
+        self.j += x
+        self.j += y
+        return self.i, self.j
```

Comment:
Can you use tests that are not associative/commutative? We cannot verify that a, b, x, y were passed in the right order here.

Reply:
updated